backend/tests/integration/migrator/test_migrator_tasks/1313624.xml

```diff
@@ -1559,10 +1559,9 @@
     <subfield code="n">arXiv:1409.0794</subfield>
     <subfield code="r"></subfield>
     <subfield code="s">2015-01-12 03:41:58</subfield>
-    <subfield code="t">arXiv</subfield>
```

Comment:
what is this?

Reply:
This record xml is used in a bunch of tests and some tests are with documents and they fail because this has source arxiv and will be hidden and not processed through our file adding. So I thought it would be easier to just remove the source arxiv instead of updating all the tests or adding a new record to test with